src/Schema/Index.php

```diff
@@ -190,7 +190,7 @@ public function spansColumns(array $columnNames)
      *
      * @return bool
      */
-    public function isFullfilledBy(Index $other)
+    public function isFulfilledBy(Index $other)
```

Comment:
This one is a BC-break, but more importantly, fulfilled is correct, isn't it?

Reply:
Yes "Fulfilled" (one initial "L" like I've got there) is the correct spelling.  I'm not great with git, but if someone can help me, I can retarget the next version instead of the current one?

Reply:
Oh sorry, I read that backwards. No need to retarget: You can introduce a new method and have the misspelt one call it for backwards-compatibility. Then, once this is merged up into 3.5.x you can deprecate the old one in favor of the new one. And finally, once that is merged up into 4.0.x, you can remove the old one.

Reply:
Sounds good - will do that momentarily.